### ISSUE TITLE
saga.attributes

### ISSUE THREAD

**Issue by mturilli** [radical-cybertools/radical.owms#3]
Shall we migrate saga.attributes into radical.utils?

**Comment by andre-merzky** [comment]
Yeah, I pondered about that, but at the end hesitated.  Point is that the exceptions raised by the attrib interface are relatively SAGA specific -- if we'd move it to util, we would need to change to plain python exceptions -- but then would need a wrapper in saga to translate back to SAGA exceptions.  But well, as it is now its also not perfect -- we eventually would need to translate the SAGA exceptions into plain python exceptions in Troy :P  Although this can probably be solved more generically with proper exception inheritance on the SAGA end.

Long story short: this probably makes sense, and will happen someday -- I am still torn on the exception rendering.  I'll leave this ticket open meanwhile...

**Comment by andre-merzky** [comment]
This is now a radical.utils ticket.  Troy itself has a very thin wrapper around attributes by now, so that an eventual migration should have minimal impact.
